doc/reference/annotations.rst

```diff
@@ -98,6 +98,36 @@ set the value via reflection, but you may change this to use a public method ins
         }
     }
 
+    class Shop
+    {
+        /**
+         * @AccessType("public_method", naming="camel_case")

```

Comment:
In the example, I think  would be more useful use this anotation at class-level.